### PR TITLE
Add standalone VNC and Rover pages

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/OS-1446) |

#### Main changes
- Web portal supports HTTPS
- Add /desktop and /rover standalone pages
- Start rover web controller using https
- Improve consistent naming: page -> tab. element -> tab

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: frontend/src/components/roverControllerLanding/__tests__/RoverControllerLanding.test.tsx [hunk 1]
```diff
@@ -236,4 +236,122 @@ describe("RoverControllerLanding", () => {
...
+    render(<RoverControllerLanding />);
+
+    expect(await screen.findByText(matchers.started)).toBeInTheDocument();
+
+    const openLink = screen.getByText(matchers.openLink, {
+      selector: "a",
+    });
+    expect(openLink).toHaveAttribute("href", "http://localhost:8070");
+    expect(openLink).toHaveAttribute("target", "_blank");
+    expect(openLink).toHaveAttribute("rel", "noopener noreferrer");
```
Comment (stefan-kreitmayer) [inline frontend/src/components/roverControllerLanding/__tests__/RoverControllerLanding.test.tsx:252]: can remove noopener as it has become redundant https://stackoverflow.com/a/50709724

File: frontend/src/pages/landingPage/LandingPage.tsx [hunk 2]
```diff
@@ -146,12 +147,12 @@ const landingPages = [
...
     ),
   },
-  {
+  rover: {
     title: "Rover Controller",
     id: "rover",
     detail: <RoverControllerLanding />,
   },
-  {
+  kb: {
```
Comment (stefan-kreitmayer) [inline frontend/src/pages/landingPage/LandingPage.tsx:155]: pls rename to "knowledge-base"
